### DISCUSSION TITLE
Feature Request: Allow specification of column width for `output-location:  column` in Revealjs

### DISCUSSION THREAD

**Discussion by pmagwene** [quarto-dev/quarto-cli#4256] [Feature Requests]
Currently, the code output location option (`output-location`) in Revealjs creates two equally sized columns for code and corresponding output.  

This often creates a sense of spatial imbalance when the output isn't nearly as wide as the code that produced it (a frequent occurence in my teaching examples).

It would be nice to be able to specify the relative width of the output column, perhaps using the same percentage based syntax as the [multiple columns option](https://quarto.org/docs/presentations/revealjs/#multiple-columns).

Proposed syntax:

````
```{r}
#| output-location: column
#| output-width: 30%
```
````

**Comment by brianmsm** [comment]
I love this `output-location: column` feature and definitely being able to control the width would be extremely useful especially in graphics where 50% is a bit short.
